src/RacialTraitSwap.cpp

```diff
@@ -447,311 +464,702 @@ class npc_race_trait_swap : public CreatureScript
 
         switch (player->GetSession()->GetSessionDbcLocale())
         {
-            case LOCALE_koKR: localizedExit = LOCALE_EXIT_1; break;
-            case LOCALE_frFR: localizedExit = LOCALE_EXIT_2; break;
-            case LOCALE_deDE: localizedExit = LOCALE_EXIT_3; break;
-            case LOCALE_zhCN: localizedExit = LOCALE_EXIT_4; break;
-            case LOCALE_zhTW: localizedExit = LOCALE_EXIT_5; break;
-            case LOCALE_esES: localizedExit = LOCALE_EXIT_6; break;
-            case LOCALE_esMX: localizedExit = LOCALE_EXIT_7; break;
-            case LOCALE_ruRU: localizedExit = LOCALE_EXIT_8; break;
+            case LOCALE_koKR:
```

Comment:
couldn't most of these switches be combined? so that e.g. localizedExit, localizedBack etc. are set within one case

Reply:
Didn't notice it lol, they can. will look at it later